doc/platform_api/CMIS_Diagnostic_Monitoring_Overview_in_SONiC.md

```diff
@@ -0,0 +1,971 @@
+    errored_frames_avg_media_input_lowwarning{lane_num}     = FLOAT          ; Errored frames average low warning value for media input
+    errored_frames_curr_media_input_highalarm{lane_num}     = FLOAT          ; Errored frames current high alarm value for media input
+    errored_frames_curr_media_input_lowalarm{lane_num}      = FLOAT          ; Errored frames current low alarm value for media input
+    errored_frames_curr_media_input_highwarning{lane_num}   = FLOAT          ; Errored frames current high warning value for media input
+    errored_frames_curr_media_input_lowwarning{lane_num}    = FLOAT          ; Errored frames current low warning value for media input
+    errored_frames_min_host_input_highalarm{lane_num}       = FLOAT          ; Errored frames minimum high alarm value for host input
+    errored_frames_min_host_input_lowalarm{lane_num}        = FLOAT          ; Errored frames minimum low alarm value for host input
+    errored_frames_min_host_input_highwarning{lane_num}     = FLOAT          ; Errored frames minimum high warning value for host input
+    errored_frames_min_host_input_lowwarning{lane_num}      = FLOAT          ; Errored frames minimum low warning value for host input
+    errored_frames_max_host_input_highalarm{lane_num}       = FLOAT          ; Errored frames maximum high alarm value for host input
+    errored_frames_max_host_input_lowalarm{lane_num}        = FLOAT          ; Errored frames maximum low alarm value for host input
+    errored_frames_max_host_input_highwarning{lane_num}     = FLOAT          ; Errored frames maximum high warning value for host input
+    errored_frames_max_host_input_lowwarning{lane_num}      = FLOAT          ; Errored frames maximum low warning value for host input
+    errored_frames_avg_host_input_highalarm{lane_num}       = FLOAT          ; Errored frames average high alarm value for host input
+    errored_frames_avg_host_input_lowalarm{lane_num}        = FLOAT          ; Errored frames average low alarm value for host input
+    errored_frames_avg_host_input_highwarning{lane_num}     = FLOAT          ; Errored frames average high warning value for host input
+    errored_frames_avg_host_input_lowwarning{lane_num}      = FLOAT          ; Errored frames average low warning value for host input
+    errored_frames_curr_host_input_highalarm{lane_num}      = FLOAT          ; Errored frames current high alarm value for host input
+    errored_frames_curr_host_input_lowalarm{lane_num}       = FLOAT          ; Errored frames current low alarm value for host input
+    errored_frames_curr_host_input_highwarning{lane_num}    = FLOAT          ; Errored frames current high warning value for host input
+    errored_frames_curr_host_input_lowwarning{lane_num}     = FLOAT          ; Errored frames current low warning value for host input
+
+### 2.9 Transceiver VDM flag data
+
+The `TRANSCEIVER_VDM_FLAG` table stores the flag status for the VDM data.
```

Comment:
@mihirpat1 should have one table for each alarm? 

TRANSCEIVER_VDM_HIGH_ALARM_FLAG
TRANSCEIVER_VDM_HWARN_ALARM_FLAG
TRANSCEIVER_VDM_LOW__ALARM_FLAG
TRANSCEIVER_VDM_LWARN_ALARM_FLAG